frontend/test/example_test.dart

```diff
@@ -33,5 +44,17 @@ void main() {
     expect(dummyComponentFixture.element.querySelector('textarea').text, contains('Hello World!'));
   });
 
-  test('New event list component is empty', () async {});
+  test('New event list component is empty', () async {
+
+    var eventServiceMock = new EventServiceMock();
+    var routerMock = new RouterMock();
+    var clientMock = new ClientMock();
+
+    var testBed = new NgTestBed<EventListComponent>().addProviders([
+      provide(EventService, useValue: eventServiceMock),
+      provide(Router, useValue: routerMock),
+      //provide(Client, useValue: clientMock),
```

Comment:
I got something like "Provider missing for Client!" error for a while. Not sure why...